operations/mimir/ruler-query-path.libsonnet

```diff
@@ -0,0 +1,80 @@
+// Deployment of an isolated, dedicated query path for ruler remote evaluation.
```

Comment:
Since in the doc we call this feature "[remote evaluation](https://grafana.com/docs/mimir/v2.1.x/operators-guide/architecture/components/ruler/#remote)", can we name the file accordingly?